src/content/feed.njk

```diff
@@ -17,7 +17,7 @@ permalink: blog/feed.xml
     <updated>{{ post.modified | convertToJsDate | dateToRfc3339 }}</updated>
     <id>{{ absolutePostURL }}</id>
     <author><name>{{ author.name }}</name></author>
```

Comment:
Auto-escaping is enabled (by default)

Reply:
I think I'm missing something about where the plugin is using `safe`.  If adding the custom `safe` filter works for the blog content don't we need to do the same for the name? Since escaping is on by default in nunjucks why doesn't that fix the problem for the content too?

Reply:
`safe` disables auto-escaping on content it is applied to (i.e. it marks the content, likely containing HTML, as safe to output as HTML)

Reply:
mm wait, I have an assumption that I didn't double check actually (which is `htmlToAbsoluteUrls()` marks the content as safe to be able to output HTML).

Reply:
I checked again and I was kinda wrong: the `htmlToAbsoluteUrls()` filter does not mark the content as safe so the output in the `feed.xml` file is escaped html but (online) feed readers probably need to unescape it to render it nicely.

Reply:
> (online) feed readers probably need to unescape it to render it nicely.

Does that make sense? I checked other parts of the site (like the list of articles) and if I remove the `safe` filter, I see the html rendered as text (escaped then).

Reply:
So if I understand correctly there would only be XSS for clients that un-escape the content that *is* currently escaped?

Reply:
That's what I understand right now but I am going to look at the spec and some other clients.

Reply:
According to the spec (https://validator.w3.org/feed/docs/atom.html#text), when we have `content` with type "html", it is expected to have entity escaped html, which is why clients will un-escape the content when they want to render it as HTML. I think it is working as intended then.

We need our `safe` plugin to sanitize the content with DOMPurify and it must be applied before `htmlToAbsoluteUrls()` (which will make the content escaped).

Reply:
I added tests.